ArduCopter/mode_plancktracking.cpp

```diff
@@ -193,6 +193,15 @@ void ModePlanckTracking::run() {
       }
     }
 
+    if (g.failsafe_planck_angle > 0){
```

Comment:
Using  `g.failsafe_planck_angle != FS_PLANCK_ANGLE_DISABLED` might be more explicit